assets/css/_swings_view.scss

```diff
@@ -8,7 +8,6 @@
   position: absolute;
   right: 0;
   top: 0;
-  z-index: 1001;
```

Comment:
Question: what's the rationale for completely removing the z-index here and in the notifications drawer? It looks like it still renders as I'd expect at least on desktop, but I've encountered a lot of weird bugs with seemingly-minor z-index changes in Skate so I just want to double-check.

Reply:
I removed them while trying to isolate the differences between these two and the vpp (vpp has the modal overlay background, which seems to make the z-index of the vpp matter more), and also noticed it didn't seem to have an impact. I'll undo this change though to limit the change for regressions

Reply:
Thanks!

Reply:
Updated $vpp-z-index to 1002 to match the original value as well.